### DISCUSSION TITLE
Applying scale and pivot on a container.

### DISCUSSION THREAD

**Discussion by a-c-sreedhar-reddy** [pixijs/pixijs#7713] [Q&A]
Hi,

I tried to apply `scale and pivot` on a container. But when I do that the container is not visible. When I apply `pivot and scale`, then the white sprite is visible as expected.

Playground link : https://www.pixiplayground.com/#/edit/v6Fq1p9ht81TOsP5VrsOX

```js

const white= PIXI.Sprite.from(PIXI.Texture.WHITE);
const container = new PIXI.Container();
container.addChild(white);
container.scale.set(15,15)
container.pivot.set(container.width/2, container.height/2)

app.stage.addChild(container);
```

**Comment by ivanpopelyshev** [comment]
container size is actually `(width,height) * getLocalBounds()` , and getLocalBounds calculates bound of container and its children in local coords.

container width and height is 0 in your case. How do you want this to work? :)

**Reply by a-c-sreedhar-reddy** [reply]
But `white` sprite which would be of (16 * 16) is added to the container.
 Then why would the container width and height be 0 ?

**Reply by ivanpopelyshev** [reply]
it will be 16. after you add that sprite.
